scripts/apply-track-config.jl

```diff
@@ -39,8 +39,20 @@ for file in global_cfg["appendable_files"]
     # Read content of append file
     append_content = read(append_path, String)
 
+    # Read org-wide-file file
+    org_wide_content = readchomp(joinpath("track-repo", file))
+
     # Append file
-    open(joinpath("track-repo", file), "a") do io
+    open(joinpath("track-repo", file), "w") do io
+        # Write org-wide content
+        write(io, org_wide_content)
+
+        # Add or remove trailing newlines depending on global config
+        # The default assumption is one trailing newline
+        newlines = haskey(global_cfg["files"], file) ? get(global_cfg["files"][file], "trailing_newlines", 1) : 1
+        write(io, "\n" ^ newlines)
+
+        # Write append content
         write(io, append_content)
```

Comment:
> So the responsibility is still on the downstream repo to end their append file with exactly one newline, so that the combined file also ends with exactly 1 newline, right? That's fine by me.

Yes. If their append file does not end with exactly one newline, they either don't care or the append file itself would cause them issues already. I don't think we have to be concerned with those two cases.